cliquet/views/errors.py

```diff
@@ -57,19 +57,28 @@ def page_not_found(request):
     redirect_enabled = request.registry.settings[config_key]
     trailing_slash_redirection_enabled = asbool(redirect_enabled)
```

Comment:
this is unrelated, but `trailing_slash_redirect_enabled` is not documented in settings docs